server/sonar-server/src/main/java/org/sonar/server/debt/DebtModelBackup.java

```diff
@@ -260,7 +259,7 @@ private static RuleDebt toRuleDebt(RuleDto rule) {
   }
 
   private void checkPermission() {
```

Comment:
yes, I already sent an email to @julienlancelot and @teryk about that. I will be dropped in another commit.